src/Microsoft.Data.SqlClient/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -10,6 +10,7 @@
 	<ItemGroup>
 	  <PackageReference Include="System.Buffers" Version="$(SystemBuffersVersion)" />
 	  <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
+    <PackageReference Include="System.Data.Common" Version="$(SystemDataCommonVersion)" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
Is this package not compatible with netcore? I know a lot of packages are still compatible with netcore but just don't do anything in scenarios where they're not needed (eg, System.Memory being part of the netcore).

Reply:
It's a netstandard1.2 package, there are no explicit .NET Core versions of it, so I thought better not mess with the .NET Core side